src/NexusMods.App.UI/Pages/Diagnostics/List/IDiagnosticListViewModel.cs

```diff
@@ -0,0 +1,13 @@
+using System.Collections.ObjectModel;
+using NexusMods.Abstractions.Loadouts;
+using NexusMods.App.UI.Controls.Diagnostics;
+using NexusMods.App.UI.WorkspaceSystem;
+
+namespace NexusMods.App.UI.Pages.Diagnostics;
+
+public interface IDiagnosticListViewModel : IPageViewModelInterface
+{
+    public LoadoutId LoadoutId { get; set; }
+
+    public ReadOnlyObservableCollection<IDiagnosticEntryViewModel> DiagnosticEntries { get; }
```

Comment:
Do we need this? Shouldn't this work like the LoadoutGrid, where we just have the LoadoutId, and get the list of mods or in this case Diagnostics form the LoadoutRegister or the DiagnosticManager?

Reply:
So I just read this wrong I think